GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -15,9 +15,10 @@ public abstract partial class POSIXPlatform : GVFSPlatform
     {
         private const string GVFSBinPath = "/usr/local/vfsforgit";
 
-        public POSIXPlatform(string installerExtension) : base(
+        public POSIXPlatform(string installerExtension, string localStorageRoot) : base(

```

Comment:
@kivikakk Are we going to need a `LinuxPlatform` class? Is that something you have in your feature branch, and can we add it to master? Or is there an existing plan for the feature integration? cc @wilbaker

Reply:
Just FYI, yes, there's a [`LinuxPlatform`](https://github.com/Microsoft/VFSForGit/tree/d3591f0eb895c13053ed06aee52f962e46e55fac/GVFS/GVFS.Platform.Linux) class coming soon to features/linuxprototype; see #986 and some of our other [WIP branches](https://github.com/github/VFSForGit/branches) in our dev fork.

Reply:
Thanks! I look forward to seeing different platform-specific pieces show up in small bites.